config/v1/types_cluster_version.go

```diff
@@ -251,6 +250,34 @@ type Update struct {
 	Force bool `json:"force"`
 }
 
+// Release represents an OpenShift release image and associated metadata.
+// +k8s:deepcopy-gen=true
+type Release struct {
+	// version is a semantic versioning identifying the update version. When this
+	// field is part of spec, version is optional if image is specified.
+	// +required
+	Version string `json:"version"`
+
+	// image is a container image location that contains the update. When this
+	// field is part of spec, image is optional if version is specified and the
+	// availableUpdates field contains a matching version.
+	// +required
+	Image string `json:"image"`
+
+	// URL describes the contents of this release and will include
+	// references to other resources as necessary.  Go here if you know
+	// of no more-specific resource addressing your question for this
```

Comment:
I don't like this sentence since it's not actionable yet.

```
URL describes the contents of the release including the appropriate change information.  This URL is set by the `url` metadata property on a release and should be displayed as a link in user interfaces. The URL field may not be set for  test or nightly releases.
```

Is both more succinct, correct, and accurate.  If we add new things, we can describe those in the future.

Reply:
`This URL is set by the `url` metadata property on a release or the metadata returned by the update API and should be displayed as a link in user interfaces.`

Reply:
Working these in in openshift/enhancements#408.

Reply:
openshift/enhancements#408 landed.  Updated the wording here to match with d5c40969 -> 575f8d2f.  Addresses [your hold][1], so:

/hold cancel

[1]: https://github.com/openshift/api/pull/521#issuecomment-662041755